app/src/test/java/com/example/mybeats/data/repository/MockedProductBodyResponse.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.mybeats.data.repository
+
+import com.example.mybeats.data.remote.model.ProductBody
+import com.example.mybeats.data.remote.model.ProductsBody
+
+object MockedProductBodyResponse {
```

Comment:
I think you don't need an object, you can just have a function or a variable, but it's up to you.

Reply:
That make senses. I thought about centralizing all possible mocks in one place, but I only used a mocked list.

I'll fix it.

Reply:
Gotcha. I'll update it.